### PR TITLE
[CloudPods] cli status command

### PR DESCRIPTION
**Pull request by MEPalma**
- Adds description of cli status command
- (Trailing spaces)

### CODE REVIEW

File: content/en/tools/cloud-pods/pods-cli/index.md [hunk 1]
```diff
@@ -104,10 +104,28 @@ Options:
   -n, --name TEXT    Name of the Cloud Pod.
   -f, --format TEXT  Format (curses, rich, json).
 ```
+
+### status
+
+The `status` command compiles a report of LocalStack current in-memory application runtime, or state, listing what CloudPod and CloudPod Version have contributed to each AWS Service.
+The set of active, or loaded, AWS Services is also given in this report so to distinguish which are not derivatives of CloudPod operations.
+
+The `-v` option extends the default report to also include the sequence of state changing CloudPod operations, listing for each the type and affected AWS Services.
```
Comment (giograno) [inline content/en/tools/cloud-pods/pods-cli/index.md:113]:
```suggestion
The `-v` option extends the default report to also include the sequence of state changing Cloud Pod operations, listing for each the type and affected AWS Services.
```

File: content/en/tools/cloud-pods/pods-cli/index.md [hunk 2]
```diff
@@ -104,10 +104,28 @@ Options:
...
+
+**Synopsis**
+```
+Lists what CloudPods have contributed to each service's current in-memory state.
```
Comment (giograno) [inline content/en/tools/cloud-pods/pods-cli/index.md:117]:
```suggestion
Lists what Cloud Pods have contributed to each service's current in-memory state.
```

File: content/en/tools/cloud-pods/pods-cli/index.md [hunk 3]
```diff
@@ -104,10 +104,28 @@ Options:
   -n, --name TEXT    Name of the Cloud Pod.
   -f, --format TEXT  Format (curses, rich, json).
 ```
+
+### status
+
+The `status` command compiles a report of LocalStack current in-memory application runtime, or state, listing what CloudPod and CloudPod Version have contributed to each AWS Service.
```
Comment (giograno) [inline content/en/tools/cloud-pods/pods-cli/index.md:110]:
```suggestion
The `status` command compiles a report of LocalStack current in-memory application runtime, or state, listing what Cloud Pod and Cloud Pod version have contributed to each AWS Service.
```

**Review by giograno — APPROVED** [review]
Thanks for also updating the docs 👍

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**